spec/features/user_creates_an_avatar_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+feature 'user signs in', %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.